src/main/java/org/broadinstitute/hellbender/cmdline/programgroups/VariantFilteringProgramGroup.java

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.hellbender.cmdline.programgroups;
+
+import org.broadinstitute.barclay.argparser.CommandLineProgramGroup;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
+
+/**
+ * Tools that filter variants by annotating the FILTER column.
```

Comment:
I'm on the fence about "by annotating the FILTER column" -- it seems overly specific and mechanism-based. I think it might hurt more than it helps. "Tools that filter variants" should be sufficient.

Reply:
Done.